BlueShift-iOS-SDK/BlueShiftAppData.m

```diff
@@ -65,8 +72,11 @@ - (NSDictionary *)toDictionary {
     }
     
     if (@available(iOS 10.0, *)) {
-        NSNumber *isPushEnabled = [NSNumber numberWithBool: self.isPushPermissionAccepted];
-        [appMutableDictionary setObject: isPushEnabled  forKey:kEnablePush];
+        NSNumber *enablePushValue = [NSNumber numberWithBool: NO];
+        if (self.enablePush == YES) {
+            enablePushValue = [NSNumber numberWithBool: self.isPushPermissionAccepted];
```

Comment:
can we have better variable names here? this piece is confusing

Reply:
Fixed